picard/ui/__init__.py

```diff
@@ -113,9 +118,21 @@ def __init__(self, parent=None):
     def keyPressEvent(self, event):
         if event.matches(QtGui.QKeySequence.Close):
             self.close()
+        elif event.matches(QtGui.QKeySequence.HelpContents) and self.help_url:
+            self.show_help()
         else:
             super().keyPressEvent(event)
 
+    def show_help(self):
+        if self.help_url:
+            url = self.help_url
+            if url.startswith('/'):
+                url = DOCS_BASE_URL + url
```

Comment:
I don't think there's any case where `DOCS_BASE_URL` could start with `goto://` so may be:

```python
if url.startswith('/'):
    webbrowser2.open(DOCS_BASE_URL + url)
elif url.startswith('goto://'):
    webbrowser2.goto(url[7:])
else:
    webbrowser2.open(url)
```